### DISCUSSION TITLE
The domain of my email does not match the domain of the university

### DISCUSSION THREAD

**Discussion by FaunoGuazina** [community/community#109636] [GitHub Education]
### Hi everyone,

I am trying to request benefits as a Student on education.github but my email is "@linkiafp.online" and my university is "Linkia.es Vocational Training", but our domains do not match, because they only use the domain - linkiafp.es - for employees.

and the discount_requests/application page does not let me continue to confirm my student status in another way, any suggestions on how I can request that or I am too dumb and I have not found a way to do it.

Thanks for the help

<img width="760" alt="Captura de pantalla 2024-02-23 a las 5 01 34" src="https://github.com/community/community/assets/53791776/4303858d-b67f-4338-bb96-c3103fbfa594">

**Comment by namith1003** [comment]
according to the GitHub maintainer in the discussion

https://github.com/orgs/community/discussions/53069

this issue happens quite often and the best thing you can do as he says in his reply is the following 

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hopefully, this answers your question, feel free to contact me if there is anything else,

A.A Namith Nimlaka
